guides/users/get-user-data.mdx

```diff
@@ -70,6 +70,33 @@ export async function userId() {
 </Tab>
 </Tabs>
 
+### Getting email from the JWT
+
+In addition to the user id you can also fetch the current email address. This includes the verification and primary status.
```

Comment:
Just tried a minor rewording here

In addition to the user ID, you can also fetch the current email address, including its verification status and whether it is set as the primary email.